value-of-pi.py

```diff
@@ -0,0 +1,2 @@
+# Python one-liner to print the value of pi
```

Comment:
Kindly update your comment with sample input and expected output.

Reply:
Done. Please review